algebra/src/main/scala/endpoints/EndpointAlg.scala

```diff
@@ -48,6 +48,24 @@ trait EndpointAlg extends UrlAlg with MethodsAlg {
     headers: RequestHeaders[C] = emptyHeaders
   )(implicit tuplerAB: Tupler.Aux[A, B, AB], tuplerABC: Tupler[AB, C]): Request[tuplerABC.Out]
 
+  /**
+    * Helper method to perform GET request
+    */
+  def get[A, C, AUnit](
+    url: Url[A],
+    headers: RequestHeaders[C] = emptyHeaders
+  )(implicit tuplerAB: Tupler.Aux[A, Unit, AUnit], tuplerABC: Tupler[AUnit, C]): Request[tuplerABC.Out] = request(Get, url, headers = headers)
```

Comment:
The `AUnit` type parameter is not useful here, you should have only two type parameters (one for the url and one for the headers).